osbuild/pipeline.py

```diff
@@ -124,13 +124,15 @@ def run(self, tree, build_tree, output_dir=None, interactive=False, check=True,
                 binds.append(f"{output_dir}:/run/osbuild/output")
                 args["output_dir"] = "/run/osbuild/output"
 
-            path = "/run/osbuild/lib" if libdir else "/usr/lib/osbuild"
+            path = "/run/osbuild/lib"
             with build_root.bound_socket("remoteloop") as sock, \
                 remoteloop.LoopServer(sock):
                 r = build_root.run(
                     [f"{path}/osbuild-run", f"{path}/assemblers/{self.name}"],
                     binds=binds,
-                    readonly_binds=[f"{tree}:/run/osbuild/tree"] + ([f"{libdir}:{path}"] if libdir else []),
+                    readonly_binds=[f"{tree}:/run/osbuild/tree"] +
+                                   ([f"{libdir}:{path}"] if libdir else [f"/usr/lib/osbuild:{path}",
+                                   f"/usr/lib/python3.7/site-packages/osbuild:{path}/assemblers/osbuild"]),
```

Comment:
CC @ondrejbudai and @larskarlitski I'd like your input here. This is clearly specific to the Python version used when building the package so we need a different approach.

Reply:
EDIT: modified based on suggestion from @ondrejbudai